packages/ui-default/templates/components/noscript_note.html

```diff
@@ -9,6 +9,10 @@
 <div class="section__body no-padding">
   <blockquote class="warn">
     <p>{{ _('This page needs JavaScript to work.') }}</p>
+    {% if handler.session.legacy and not handler.session.nohint %}
```

Comment:
This hint will only display on those page that can't be used when javascript is not available.
But for most pages, the user will get a downgraded experience without this hint. (Additional functionality will be disabled)
It's would be much better to add a hint globally.